whois-query/src/main/java/net/ripe/db/whois/query/acl/HazelcastPersonalObjectAccounting.java

```diff
@@ -74,18 +84,53 @@ public int accountPersonalObject(final InetAddress remoteAddress, final int amou
         return 0;
     }
 
-    private void unlockKey(InetAddress remoteAddress) {
+    @Override
+    public int accountPersonalObject(final String ssoId, final int amount) {
+        boolean isLocked = false;
```

Comment:
Since the map is different and its signature as well so I preferred to do it separately as there is not much logic in it mostly putting an entry in map